automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -25,7 +25,7 @@ import spray.json._
 import DefaultJsonProtocol._
 
 import scala.concurrent.duration._
-
+import scala.language.postfixOps
```

Comment:
this is not necessary for this PR, but I was watching compile errors/warnings due to the dependency upgrade, and I made this fix, which eliminates a compile warning.